src/System.Net.Sockets.Legacy/ref/System.Net.Sockets.Legacy.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+
+namespace System.Net.Sockets
+{
+   
+    public static class Legacy
+    {
+        public static System.IAsyncResult BeginAccept(this System.Net.Sockets.Socket thisSocket, System.AsyncCallback callback, object state) { return default(System.IAsyncResult); }
```

Comment:
You can use our GenAPI tool to generate these from the library to help ensure they exactly match your implementation.